resources/scss/mixins.scss

```diff
@@ -0,0 +1,50 @@
+@mixin item-block {
+  padding: 15px;
+  background: #fff;
+  overflow: hidden;
+}
+
+@mixin clearfix {
+  clear: both;

```

Comment:
clearfix should have other properties as well

Reply:
done